assets/scripts/app/App.jsx

```diff
@@ -80,4 +84,7 @@ function mapStateToProps (state) {
   }
 }
 
-export default connect(mapStateToProps)(App)
+export default flow(
+  connect(mapStateToProps),
+  DragDropContext(HTML5Backend)
+)(App)
```

Comment:
I'm not familiar with this pattern but I understand what it's doing -- just curious if this is something explained in the react-dnd docs?

Reply:
Not exactly. I wasn't quite sure how to connect the component to Redux and connect it to the react-dnd so I looked it up and found this: https://github.com/react-dnd/react-dnd/issues/461